base/reduce.jl

```diff
@@ -938,11 +938,10 @@ end
 _bool(f) = x->f(x)::Bool
 
 """
-    count(p, itr) -> Integer
-    count(itr) -> Integer
+    count([f=identity,] itr; init=0) -> Integer
 
-Count the number of elements in `itr` for which predicate `p` returns `true`.
-If `p` is omitted, counts the number of `true` elements in `itr` (which
+Count the number of elements in `itr` for which the function `f` returns `true`.
+If `f` is omitted, counts the number of `true` elements in `itr` (which
```

Comment:
The `init` keyword should be documented :) Also, what was wrong with "predicate"?

Reply:
Honestly, I don't quite remember, why I changed this... :laughing: I think I changed the name from `p` to `f` to be more consistent with oder `mapreduce`-like functions, so calling it "function" ended up making more sense? `function` does seem clearer to me here, because that's the terminology we use throughout the manual, and it seems a bit unnecessarily confusing to call it something else here. I don't feel very strongly about this though, so I can also leave it as-is.

Reply:
Oh, one last small bit:
```suggestion
If `f` is omitted, count the number of `true` elements in `itr` (which
```